conftest.py

```diff
@@ -0,0 +1,151 @@
+import builtins
+from contextlib import contextmanager
+from pathlib import Path
+import yaml
+from typing import Callable
+
+import pytest
+
+
+# Prevent errors caused by pytest trying to collect tests from externals 
+# (e.g. from errors whilst importing python code in the UCLA-roms repo, see https://github.com/CWorthy-ocean/C-Star/issues/53)
+collect_ignore_glob = [
+    "cstar/cstar_local_config.py",
+    "cstar/externals/**"
+]
+
+
+EXAMPLE_BLUEPRINTS = {
+    "ROMS_MARBL": {
+        # TODO this assumes you are running pytest from the root directory of the cloned repo, which is fragile
+        "base": "./examples/cstar_blueprint_roms_marbl_example.yaml",
+        "input_datasets_url_prefix": "https://github.com/CWorthy-ocean/input_datasets_roms_marbl_example/raw/main/",
+        "additional_code_url": "https://github.com/CWorthy-ocean/cstar_blueprint_roms_marbl_example.git",
```

Comment:
Do we actually have to ship these additional urls along with the blueprint? Or can they always be determined from the blueprint?

Reply:
they can be determined from the blueprint

Reply:
Okay. But how? I mean I see that that `input_datasets_url_prefix` exists as a substring of the `location` fields of each of the files under `input_datasets`. But is that actually a parameter than I must derive? Or can I just say "any input dataset file needs to have its location changed to be local", in which case I don't actually need to know what the url prefix is in order to modify that paths.